src/PackageClassifier/ClassificationEntry.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace PackageClassifier
+{
+    public class ClassificationEntry
+    {
+        public ClassificationEntry(string pattern, Trait [] traits)
+        {
+            Pattern = pattern;
+            Traits = traits;
+        }
+
+        public string Pattern { get; }
+
+        public Trait [] Traits { get; }
+
+        public bool HasPattern(string pattern)
```

Comment:
Could we remove this for now? https://github.com/aspnet/Coherence/blob/dev/packages/packages.csv doesn't have * (which are problematic regardless).